src/pages-and-resources/discussions/data/api.js

```diff
@@ -46,6 +46,7 @@ function normalizePluginConfig(data) {
 function normalizeApps(data) {
   const apps = Object.entries(data.providers.available).map(([key, app]) => ({
     id: key,
+    messages: app.messages || {},
```

Comment:
by using namedtuple with defaults, `app.messages` will never be set to undefined or empty, so why are you using `|| {}`?.

Reply:
Any thoughts on this ^^

Reply:
I removed the fallback because the field is always guaranteed to be non-null.